test/inductor/test_torchinductor.py

```diff
@@ -13741,6 +13742,50 @@ def forward(
                 )
                 torch._inductor.aot_compile(traced, inputs)
 
+        @skipCUDAIf(not SM90OrLater, "Requires sm90")
+        @requires_gpu()
+        @config.patch(implicit_fallbacks=True)
+        def test_grouped_mm(self):
+            @torch.compile(fullgraph=True)
+            def f(a, b, offs, out_dtype):
+                return torch._grouped_mm(
+                    a, b.transpose(-2, -1), offs=offs, out_dtype=out_dtype
+                )
+
+            device = "cuda"
```

Comment:
Hi, may I suggest we replace the hardcode '"cuda"' with `GPU_TYPE`? This test case is decorated with `requires_gpu` and will also runs on Intel GPU, which will fail with cuda not compiled error.
You can also use `requires_cuda` here.

Reply:
I'm going to come back to this PR in a few days but good call out, will update